mycore-base/src/test/java/org/mycore/datamodel/objectinfo/MCRObjectInfoEntityQueryResolverTest.java

```diff
@@ -88,8 +89,8 @@ private static void initClassifications() throws SAXParseException, MalformedURL
         DAO = new MCRCategoryDAOImpl();
         CLS = MCRCategLinkServiceFactory.getInstance();
 
-        Document xml = MCRXMLParserFactory.getParser()
-            .parseXML(new MCRURLContent(new URL("https://mycore.de/classifications/mir_licenses.xml")));
+        URL classResourceUrl = MCRCategoryDAOImplTest.class.getResource("/mycore-classifications/mir_licenses.xml");
```

Comment:
This still uses the wrong class object. `MCRObjectInfoEntityQueryResolverTest` should use `MCRObjectInfoEntityQueryResolverTest.class` as a reference when loading resources.

Reply:
Should be fixed by [22c5cc0](https://github.com/MyCoRe-Org/mycore/pull/1757/commits/22c5cc0f9b20ae08c66d4337059e33b1d631059d)

Reply:
Codacy dosn't like the change though, ...